tests/merge_sorted_lists_test.py

```diff
@@ -1,2 +1,37 @@
 import pytest
 from functions.merge_sorted_lists import merge_sorted_lists
+
```

Comment:
I think your tests are currently failing or erroring out. Try some further debugging and then I can re-review.

Reply:
Correct! Let me take a look.